scripts/build.js

```diff
@@ -132,14 +132,31 @@ function build(options) {
       process.exit(1);
     }
 
-    var args = [require.resolve(path.join('node-gyp', 'bin', 'node-gyp.js')), 'rebuild', '--verbose'].concat(
-      ['libsass_ext', 'libsass_cflags', 'libsass_ldflags', 'libsass_library'].map(function(subject) {
+    var args = ['rebuild', '--verbose'].concat(
+      ['libsass_ext', 'libsass_cflags', 'libsass_ldflags', 'libsass_library'].map(function (subject) {
         return ['--', subject, '=', process.env[subject.toUpperCase()] || ''].join('');
       })).concat(options.args);
 
-    console.log(['Building:', process.execPath].concat(args).join(' '));
+    // To make node-chakracore work, check if node-gyp is in the path.
+    // If yes, use it instead of using node-gyp directly from node_modules because it might not be
```

Comment:
I find this comment to be rather misleading since it is only true if we're running under chakracore.